polishNotation.py

```diff
@@ -0,0 +1,39 @@
+def polishNotation(expression):
+    s = stack()
+    result = 0

```

Comment:
|result| should be kept at the local scope of the different operators below. Making its scope at the function level just confuses the reader about its use.

Note that this makes no difference for python as the scope of variables is a function: https://en.wikipedia.org/wiki/Scope_(computer_science)#Python

In other languages (C,C++,Java), the compiler allows narrower scope, which prevents using a variable outside of its intended scope.